.github/workflows/codeql-analysis.yml

```diff
@@ -13,15 +13,17 @@ name: "CodeQL"
 
 on:
   push:
+    branches: [ main ]
   pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
   schedule:
     - cron: '15 17 * * 5'
 
 jobs:
   analyze:
     name: Analyze
     runs-on: ubuntu-latest
-    if: ${{ github.event_name != 'pull_request' || github.repository_owner != github.event.pull_request.head.repo.owner.login }}
```

Comment:
We don't need to revert this, as this is a little different to the renovate repo, here i removbed the branch filter, so it currently works as expected.

in main repo i didn't removed the branch filter, so codeql scan did never run for pr there.

So if we revert this, this worflow will only on on schedule, on main branch and pr's targeting main.